src/System.Private.ServiceModel/tests/Common/Infrastructure/src/CertificateManager.cs

```diff
@@ -204,5 +167,18 @@ public static X509Certificate2 InstallCertificateToMyStore(X509Certificate2 cert
 
             return certificate;
         }
+
+        // Install the certificate into the TrustedPeople store.
+        // It will not install the certificate if it is already present in the store.
+        // It returns the thumbprint of the certificate, regardless whether it was added or found.
+        public static X509Certificate2 InstallCertificateToTrustedPeopleStore(X509Certificate2 certificate)
```

Comment:
Reviewers: for PeerTrust to work, it is necessary that the server's certificate be installed on the client into the TrustedPeople store for CurrentUser.  This is the code that gets called by the ConditionalFact for Server_Certificate_Installed.  It is also validated by new scenario tests.